### ISSUE TITLE
Support CUDA 9.1 on Windows

### ISSUE THREAD

**Issue by maleadt** [JuliaGPU/CUDAapi.jl#36]
Maybe there's something wrong with how CI (hackishly) installs CUDA, but it looks like CUDA 9.1 isn't detected properly.

**Comment by blairn** [comment]
How can I test if it is being detected? I'm trying to install Knet, and having trouble, and was pointed here.

**Comment by maleadt** [comment]
Well, what are you trying to discover with CUDAapi? From the logs over at https://github.com/denizyuret/Knet.jl/issues/285, it looks like CUDAapi is working properly:

```
WARNING: Found multiple CUDA toolkit installations: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1 and C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common
```

It doesn't seem to detect CUDNN though, `Could not find cudnn library`, but that's not what this issue is about. Do you even have CUDNN installed?

Run `DEBUG=1 julia --compilecache=no -e 'Pkg.build("Knet")'` and post the resulting output (and please use quote blocks).

**Comment by blairn** [comment]
Thank you! It was a bad CUDNN install, I thought that having CUDA working meant CUDnn was also working.

Here is a Haiku for the help (I think your name has 2 syllables).

Windows CUDA fault
Installs now fixed issues
Maledt's knowledge win

**Comment by maleadt** [comment]
Closed via 158ef4582f11b183ec3bdcc9323b43dc5253ca4b.
